src/main/java/com/openlattice/chronicle/services/twilio/TwilioService.kt

```diff
@@ -0,0 +1,59 @@
+package com.openlattice.chronicle.services.twilio
+
+import com.openlattice.chronicle.configuration.TwilioConfiguration
+import com.openlattice.chronicle.constants.MessageDetails
+import com.openlattice.chronicle.constants.MessageOutcome
+import com.twilio.Twilio
+import com.twilio.exception.ApiException
+import com.twilio.rest.api.v2010.account.Message
+import com.twilio.type.PhoneNumber
+import org.slf4j.LoggerFactory
+import java.net.URI
+import java.time.OffsetDateTime
+
+/**
+ * @author toddbergman &lt;todd@openlattice.com&gt;
+ */
+class TwilioService(configuration: TwilioConfiguration) :
+    TwilioManager {
+    private val fromPhoneNumber: PhoneNumber = PhoneNumber(configuration.fromPhone)
+    override fun sendMessage(participantId: String, messageDetails: MessageDetails): MessageOutcome {
+        val messageText = "Follow this link to enroll in Chronicle: {{URL}}".replace(URL, messageDetails.url)
+        return try {
+            val message = Message
+                .creator(PhoneNumber(messageDetails.phoneNumber), fromPhoneNumber, messageText)
+                .setStatusCallback(
+                    URI
+                        .create("https://api.openlattice.com/bifrost/messages/status")
+                )
+                .create()
+            MessageOutcome(
+                messageDetails.messageType,
+                OffsetDateTime.now(),
+                participantId,
+                messageDetails.url,
+                message.status != Message.Status.FAILED,
```

Comment:
More intuitive to have status be of type Message.Status instead of boolean

Reply:
is it now binary? It will be successful or failed, which seems appropriate for a boolean value.

Reply:
super picky here, but it is hard to reason about true/false being suitable responses to the question: "What is the status of the message?".

Reply:
ok ok ok

Reply:
to be clear - the field on the entity is not status. It is `Delivered (Y/N)`. 

would you like me to add a status field to the message entity?

Reply:
boolean is good enough for now